### PR TITLE
CLI - first draft

### PR DESCRIPTION
**Pull request by TimHenkelmann**
This is the first draft of the CLI. 
Notice that this still has a few issues but should provide a template such that Team 1 can work on this together.
Obviously, this won't go into prod with all those comments!

### CODE REVIEW

**Review by ShilpaGhanashyamGore — APPROVED** [review]
Yup .We are going to build on this .
Lets merge the first version.

File: testscripts/test3.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/**
+ * Created by TimH on 28-Mar-17.
```
Comment (kordianbruck) [inline testscripts/test3.js:2]: :(

Comment (TimHenkelmann) [reply testscripts/test3.js:2]: Those are only scripts for testing purposes. They get removed once everything works

Comment (kordianbruck) [reply testscripts/test3.js:2]: I know - still :stuck_out_tongue:

Comment (TimHenkelmann) [reply testscripts/test3.js:2]: true https://m.popkey.co/6bee24/6GJWk.gif

**Pull request merged**